GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -263,16 +261,24 @@ private static object ConvertReturnToDownloadFormat(Return returnForYear)
             };
         }
 
-        [HttpGet("downloads-new/late-submissions")]
-        public FileContentResult DownloadLateSubmissions()
+        [HttpGet("downloads-new/late-submissions-for-{year}")]
+        public FileContentResult DownloadLateSubmissions(int year)
         {
             List<Organisation> organisationsWithLateReturns = dataRepository.GetAll<Organisation>()
                 .Where(org => org.Status == OrganisationStatuses.Active)
                 .Where(
-                    org => org.LatestScope == null
-                           || org.LatestScope.ScopeStatus == ScopeStatuses.InScope
-                           || org.LatestScope.ScopeStatus == ScopeStatuses.PresumedInScope)
-                .Where(org => org.LatestReturn == null || org.LatestReturn.IsLateSubmission)
+                    org => org.OrganisationScopes.Any(
+                        s => s.SnapshotDate.Year == year
+                             && (s.ScopeStatus == ScopeStatuses.InScope || s.ScopeStatus == ScopeStatuses.PresumedInScope)))
```

Comment:
Could you explain this one to me a bit further please? Does it still cover the `org.LatestScope == null` case from line 272?

Reply:
Good spot - it doesn't cover that case - I'll fix that